apps/aecore/lib/aecore/structures/spend_tx.ex

```diff
@@ -3,43 +3,129 @@ defmodule Aecore.Structures.SpendTx do
   Aecore structure of a transaction data.
   """
 
-  alias Aecore.Structures.SpendTx
+  @behaviour Aecore.Structures.Transaction
 
-  @type t :: %SpendTx{
-    from_acc: binary(),
+  alias Aeutil.Serialization
+  alias Aeutil.Parser
+  alias Aecore.Structures.Account
+
+  @typedoc "Arbitrary structure data of a transaction"
+  @type payload :: %__MODULE__{} | map()
```

Comment:
can you be more specific of the `map()` type and general about what can be "arbitrary" exactly